tests/functional/i/iterating_modified.rc

```diff
@@ -0,0 +1,2 @@
+[Messages Control]
+disable=locally-disabled,useless-suppression,missing-module-docstring
```

Comment:
Can you add the necessary disables to the test file itself? You can add a `pylint: disable` line after the module docstring. We place disables there by convention in the test files so we know what is disabled for each test when opening the file.

Reply:
The rcfiles in functional tests are new for me.. I am actually used to the old method like you said with:

` # pylint disable=`

So what should I put in the rc file?

Reply:
> I would change the test but what should I put in the rc file?

The rc file is optional, so you can simply delete it. You only need it if there are other settings required for your test (an example would be if your PR changed the default evaluation formula and you needed to test a file with a different evaluation formula).